pygeoapi_iwls/s104.py

```diff
@@ -37,15 +37,24 @@ def _get_flags(self,x):
     def _gen_S104_trends(self,df_wl):
         """
         Generate water level trend flags from water level values
-        :param df_wl: pandas dataframe containing water level values
-        :return: pandas Dataframe containing trend Flags for respective water level values
+        :param df_wl: pandas dataframe containing water level values (pandas dataframe)
+        :return: pandas Dataframe containing trend Flags for respective water level values (pandas dataframe)
         """
         if not df_wl.empty:
             df_wl_trend = df_wl
             interval = (df_wl_trend.index[1] - df_wl_trend.index[0]).total_seconds()
             timestamps_per_hour = int(3600 // interval)
+            # Create mask for NaN values
```

Comment:
Incorporated suggested changes in commit in latest commit. I've used isna() instead of isnull() for consistency wihtt he rest of the method. (isnull() is an alias for isna())